kpi/permissions.py

```diff
@@ -120,7 +120,7 @@ class AssetNestedObjectPermission(BaseAssetNestedObjectPermission):
     perms_map['HEAD'] = perms_map['GET']
     perms_map['PUT'] = perms_map['POST']
     perms_map['PATCH'] = perms_map['POST']
-    perms_map['DELETE'] = perms_map['POST']
+    perms_map['DELETE'] = perms_map['GET']
```

Comment:
:ok_hand: we can get away with this because of
https://github.com/kobotoolbox/kpi/blob/73271f9bc2077c4cf5b873dc1fe522030d2c5ba3/kpi/utils/object_permission_helper.py#L60-L65